config/notifiers.go

```diff
@@ -92,6 +92,17 @@ var (
 		// TODO: Add a details field with all the alerts.
 	}
 
+	// DefaultVictorOpsConfig defines default values for VictorOps configurations.
+	DefaultVictorOpsConfig = VictorOpsConfig{
+		NotifierConfig: NotifierConfig{
+			VSendResolved: true,
+		},
+		MessageType:  `{{ if eq .Status "firing" }}CRITICAL{{ else }}RECOVERY{{ end }}`,
```

Comment:
I wonder whether or not this MessageType switching should occur instead impl.go.  Kind of like how the pagerduty and opsgenies ones do it. (PagerDuty): https://github.com/rhazdon/alertmanager/blob/001e0716bda50a9721b7c8b9cc32751969f6b8e6/notify/impl.go#L405 

But I can see an argument against it if we want to be able to use INFO, WARNING, etc.